app/scripts/components/common/page-footer/styles.scss

```diff
@@ -48,18 +48,25 @@
   padding: themeVars.$veda-uswds-padding-4 themeVars.$veda-uswds-padding-5;
   background-color: themeVars.$veda-uswds-color-base-lightest;
 }
-#footer_secondary_container {
-  padding: themeVars.$veda-uswds-padding-4 themeVars.$veda-uswds-padding-5;
-}
 
 .usa-footer__secondary-section {
   background-color: themeVars.$veda-uswds-color-base-darkest;
+  padding: themeVars.$veda-uswds-padding-4 themeVars.$veda-uswds-padding-5;
+  & > .grid-container {
+    max-width: 100%;
+    padding: 0;
+    //NEED to determine how to remove 64rem max-width on grid-container
```

Comment:
why do we need to remove the max-width? This is the desktop breakpoint afaik

Reply:
Correct, the existing footer function when at desktop view expands to the full width of the entire window. With this USWDS max-width in place it only allows for it to stretch to 64rem and leaves pretty large gutters on either side:  Example: https://trussworks.github.io/react-uswds/iframe.html?args=&id=components-footer--slim-footer&viewMode=story

This might be more of a @faustoperez question. At desktop do we want the [large gutter to the left and right](https://trussworks.github.io/react-uswds/iframe.html?args=&id=components-footer--slim-footer&viewMode=story) of the footer content at desktop or do we want the content to stretch to[full width of the screen](https://deploy-preview-1308--veda-ui.netlify.app/)?

Reply:
IMO we should follow the header and leave the gutter. If I understand it correctly we wil have a page layout later that will align all the components on a page.